<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnFMultitool</class>
 <widget class="QWidget" name="SnFMultitool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnFMultitool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/appIcon/resources/snfLogo.png</normalon>
   </iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>129</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>28</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/fileOpen/D:/Descargas/file_open_FILL0_wght400_GRAD0_opsz48.png</normalon>
    </iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>-17</x>
     <y>-10</y>
     <width>651</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="snfmultitool.qrc"/>
 </resources>
 <connections/>
</ui>
